homeassistant/components/frontend/__init__.py

```diff
@@ -81,7 +94,26 @@
         vol.Optional(CONF_EXTRA_HTML_URL_ES5):
             vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(CONF_JS_VERSION, default=JS_DEFAULT_OPTION):
-            vol.In(JS_OPTIONS)
+            vol.In(JS_OPTIONS),
+        vol.Optional(CONF_MANIFEST_JSON, default={
+            CONF_MANIFEST_JSON_DISPLAY: CONF_MANIFEST_JSON_DISPLAY_DEFAULT,
+            CONF_MANIFEST_JSON_SHORT_NAME:
+                CONF_MANIFEST_JSON_SHORT_NAME_DEFAULT,
```

Comment:
I've been thinking about this, I wonder if instead of short name default, we should default to `hass.config.location_name`

Reply:
I considered this too but my assumption is that most users (me included) just set that to something general like 'Home' and this may cause a lot of confusion when you see an app called 'Home' on your phone or tablet. This setting was included in the user customizable ones because I read some community forum posts while researching manifest.json in Home Assistant in which users wanted to change to shortcut name from 'Assistant' to 'Home Assistant' just because it was causing confusion. I think most users just want to see the application name because it is more familiar.

Reply:
The problem with "Home Assistant" is that it will be cut off and you get ellipsis in the name :( and it will become `Home Ass…`. 

The thing is, I am not a fan of config options. They have to be maintained forever, and all permutations have to be checked.

Reply:
I am not suggesting changing the default name to Home Assistant. And I agree with you don;t give the user too many cihoices because they will not know what to choose.